src/index.js

```diff
@@ -5,18 +5,24 @@ import { makeServer } from "./server";
 import { ProductContextProvider } from "./context/products/productContext";
 import { BrowserRouter } from "react-router-dom";
 import { CartContextProvider } from "./context/cart/cartContext";
+import { WishlistContextProvider } from "./context/wishlist/wishlistContext";
+import { AuthContextProvider } from "./context/auth/authContext";
 // Call make Server
```

Comment:
you can try importing all files from `./context`

Reply:
I will do